cephfs/file.go

```diff
@@ -23,6 +23,17 @@ const (
 	SeekEnd = int(C.SEEK_END)
 )
 
+// SyncChoice is used to control how metadata and/or data is sync'ed to
+// the file system.
+type SyncChoice int
+
+const (
+	// SyncMetaData will synchronize both data and metadata.
+	SyncMetaData = SyncChoice(0)
```

Comment:
This name suggests only metadata (yeah, I noticed **M**eta**D**ata too) is synced. The documentation for `ceph_fsync()` says the argument is a boolean, why not just pass a `syncdataonly` parameter as bool?

Reply:
I can try to tweak the name. How about `SyncAll` ? Alternatively I could go with the more explicit `SyncDataMetaData` but seems kind of long and redundant.

I don't want to use a bool, it prevents future extensibility and doesn't match as closely with the underlying ceph api. Plus an expelict "enum" is more self documenting IMO. What is more obvious on first glance: `foo.Fsync(SyncDataOnly)` or `foo.Fsync(true)`?

Reply:
`SyncAll` has my preference. `ceph_fsync()` documents the parameter as a boolean called `syncdataonly`, so I think a boolean is clear enough. Golang docs would also mention the name of the variable in the parameter list of the function?